packages/splits-sdk/src/client/base.ts

```diff
@@ -92,24 +56,23 @@ class BaseClient {
     this._publicClient = publicClient
     this._chainId = chainId
     this._walletClient = walletClient
-    this._graphqlClient = getGraphqlClient(chainId)
     this._includeEnsNames = includeEnsNames
-  }
 
-  protected async _makeGqlRequest<ResponseType>(
-    query: string,
-    variables?: Variables,
-  ): Promise<ResponseType> {
-    if (!this._graphqlClient) {
-      throw new UnsupportedSubgraphChainIdError()
+    if (apiConfig) {
+      this._dataClient = new DataClient({
+        publicClient,
+        ensPublicClient,
+        apiConfig,
+        includeEnsNames,
+      })
     }
+  }
 
-    // TODO: any error handling? need to add try/catch if so
-    const result = await this._graphqlClient.request<ResponseType>(
-      query,
-      variables,
-    )
-    return result
+  protected _requireDataClient() {
+    if (!this._dataClient)
+      throw new MissingDataClientError(
+        'Data client required to perform this action, please update your call to the constructor',
```

Comment:
nit: Should this say api key required? is that the actual change the dev would need to make?